hypertrace-graphql-gateway-service-metric-utils/src/main/java/org/hypertrace/graphql/utils/metrics/gateway/MetricContainerMapConverter.java

```diff
@@ -71,13 +70,13 @@
         ConvertedMetricContainer::new);
   }
 
-  private static class ConvertedMetricContainer extends ConvertedAggregationContainer
+  private static class ConvertedMetricContainer extends BaselineConvertedAggregationContainer
       implements MetricContainer {
 
     private final Map<Duration, List<MetricInterval>> metricSeriesMap;
 
     ConvertedMetricContainer(
-        Map<MetricLookupMapKey, MetricAggregation> metricAggregationMap,
+        Map<MetricLookupMapKey, BaselineMetricAggregation> metricAggregationMap,
```

Comment:
BaselineMetricAggregation implies something totally different from the existing MetricAggregation. Are you sure you want to do a replace/rename instead of an augmentation to the existing users of MetricAggregation with a different object of the type BaselineMetricAggregation?